### PR TITLE
Added Portuguese language

### PR DESCRIPTION
**Pull request by smarquespt**
Anything else I need to wok on?

### CODE REVIEW

File: _Build/_Language/_Language-patches/pt-PT.browser.dtd [hunk 1]
```diff
@@ -0,0 +1,219 @@
...
+<!-- Purge RAM browser tweak -->
+<!ENTITY minimizeMemoryCache.label "Limpar cache">
+<!ENTITY turnOnCleanRam.label "Ativa o item de menu Limpar cache.">
+<!-- End Purge RAM browser tweak -->
+
+<!-- Restart Browser Tweak -->
+<!ENTITY restartBrowser.label "Reiniciar navegador">
+<!ENTITY restartBrowserButton.label "Reiniciar navegador">
+<!ENTITY turnOnRestart.label "Ativa o item de menu Reiniciar navegador.">
+<!ENTITY cleanFastRestartCache.label "Clear fast restart cache on browser restart.">
```
Comment (smarquespt) [inline _Build/_Language/_Language-patches/pt-PT.browser.dtd]: What does this do exactly?

File: _Build/_Language/_Language-patches/pt-PT.browser.dtd [hunk 2]
```diff
@@ -0,0 +1,219 @@
...
+<!ENTITY restartOptions.label "Restart Options">
```
Comment (smarquespt) [inline _Build/_Language/_Language-patches/pt-PT.browser.dtd]: To Restart Options (reset) or the Options of Restart?

Comment (InternalError503) [inline _Build/_Language/_Language-patches/pt-PT.browser.dtd]: Restart Options as in a group of options for the restart feature, Here is a image
c5rv6byhunjimko

**Comment by InternalError503** [comment]
Thanks so much @smarquespt i will push this in and apply the current to both beta and release, Again thank you.

File: _Build/_Language/_Language-patches/pt-PT.browser.dtd (hunk 2 above)
Comment (smarquespt) [inline _Build/_Language/_Language-patches/pt-PT.browser.dtd]: Thanks. Fixed in https://github.com/InternalError503/cyberfox/pull/2

**Pull request merged**